tools/tf2openapi/generated/framework/remote_fused_graph_execute_info.pb.go

```diff
@@ -32,7 +32,7 @@ type RemoteFusedGraphExecuteInfo struct {
 	GraphOutputNodeName []string `protobuf:"bytes,3,rep,name=graph_output_node_name,json=graphOutputNodeName,proto3" json:"graph_output_node_name,omitempty"`
 	// Executor's name
 	ExecutorName string `protobuf:"bytes,4,opt,name=executor_name,json=executorName,proto3" json:"executor_name,omitempty"`
-	// Optional: Parameters given to the executor
+	// Optional: Parameters given to the inference-logger
```

Comment:
lol! something wrong here?

Reply:
Not sure how that got there. Anyway, I have rerun the Makefile to regenerate the files.